server/tools/docker-entrypoint.sh

```diff
@@ -24,6 +24,18 @@ if [ "$KEYCLOAK_HOSTNAME" != "" ]; then
     fi
 fi
 
+########################
+# JGroups bind options #
+########################
+
+if [ -z "$BIND" ]; then
+    BIND=$(hostname -i)
+fi
+if [ -z "$BIND_OPTS" ]; then
+    BIND_OPTS="-Djboss.bind.address.management=0.0.0.0 -Djgroups.join_timeout=1000 -Djgroups.bind_addr=$BIND -Djboss.bind.address=$BIND"
```

Comment:
The `jgroups.bind_addr`  has effect in WildFly-based software using jgroups subsystem, which I assume is the case here.